src/components/DateRelated.tsx

```diff
@@ -1,33 +1,39 @@
 import React from 'react';
 import { StyleSheet, View, Text } from 'react-native';
 import colors from '@styles/colors';
-import { format } from 'date-fns';
+import { format, getDay } from 'date-fns';
 import { ru } from 'date-fns/locale';
 import { WeatherIcon, WeatherType } from '@components/WeatherIcon';
 
-interface IProps {
-    dayOfTheWeek: string;
+export interface IProps {
     date: string;
     weatherType: WeatherType;
-    temperatureAfternoon: number;
-    temperatureNight: number;
+    dayTemp: number;
+    nightTemp: number;
 }
 
-const getWeek = format(new Date(), 'cccc', { locale: ru });
+const getWeekDay = (incomingDate: string) => {
+    const currentDate = new Date();
+    const weekNumberFromApi = getDay(new Date(incomingDate)).toString();
+    const currentWeekNumber = getDay(currentDate).toString();
+    return weekNumberFromApi === currentWeekNumber
+        ? 'Сегодня'
+        : format(new Date(weekNumberFromApi), 'cccc', { locale: ru });
```

Comment:
There is isSameDay function in date-fns.
![image](https://user-images.githubusercontent.com/52620425/224565633-d37f63d6-e7e7-49d0-9c3f-f47a58cdf114.png)
new Date(number) will return wrong info. You better test everything before push,
Place string in constants